### PR TITLE
Add RMT localization test for Chinese

### PR DESCRIPTION
**Pull request by lemon-suse**
To improve our test coverage, we will add the automation test of RMT localization test for Chinese.

- Related ticket: https://progress.opensuse.org/issues/101328
                        https://progress.opensuse.org/issues/101325
- Needles: Already integrated into OSD.
- Verification run: https://openqa.nue.suse.com/tests/7704169#step/rmt_chinese/91
  To avoid regression: https://openqa.nue.suse.com/tests/7704184#live  # rmt_feature test
                                https://openqa.nue.suse.com/tests/7704415#  # offline pscc test to verify no regression for 
                                  handle_login code update

### CODE REVIEW

**Comment by chcao** [comment]
Need to mark these two needles as workaround:
https://openqa.nue.suse.com/tests/7704169#step/rmt_chinese/86
https://openqa.nue.suse.com/tests/7704169#step/rmt_chinese/89
There's a known bug: bsc#1127676

**Comment by tinawang123** [comment]
You did not create new common function for rmt wizard?

**Comment by lemon-suse** [comment]
New verification log https://openqa.nue.suse.com/tests/7704442# for the update code and feedback of Chenzi.

**Comment by lemon-suse** [comment]
> You did not create new common function for rmt wizard?

In fact, I have think about to create a new function for it, but later I think it will cause so many duplicate code, at last I decided to update the rmt_wizard function.

**Review by chcao — APPROVED** [review]
LGTM

File: tests/x11/rmt/rmt_chinese.pm
Comment (GraceWang571) [inline tests/x11/rmt/rmt_chinese.pm:54]: > handle_logout;
> handle_login;

The codes can be replaced by `handle_relogin;`

**Review by GraceWang571 — CHANGES_REQUESTED** [review]
This is looking good. 
I left a comment that should be addressed before this gets merged.

**Comment by GraceWang571** [comment]
One more comment about the naming of the new needle. The needle name should be updated in order not to mislead users.
Actually, the newly added needle is not for `application_starts_on_login` right?
https://openqa.nue.suse.com/tests/7704169#step/rmt_chinese/64

File: lib/repo_tools.pm
Comment (GraceWang571) [inline lib/repo_tools.pm:253]: Do we really need to `record_soft_failure('bsc#1191112 - When navigating through YaST module screens the next screen appears, but its content is not loaded');` twice here?

File: tests/x11/rmt/rmt_chinese.pm
Comment (lemon-suse) [reply tests/x11/rmt/rmt_chinese.pm]: Ok, thanks.Verified with https://openqa.nue.suse.com/tests/7714601#.

File: lib/repo_tools.pm
Comment (lemon-suse) [reply lib/repo_tools.pm:253]: Yes, from the recent test it will blocked by the bug frequently so we need add this workaround. The two places are different so need add more workaround.

**Comment by lemon-suse** [comment]
> One more comment about the naming of the new needle. The needle name should be updated in order not to mislead users. Actually, the newly added needle is not for `application_starts_on_login` right? https://openqa.nue.suse.com/tests/7704169#step/rmt_chinese/64

Updated needle, see https://openqa.nue.suse.com/tests/7714601#step/rmt_chinese/64

**Comment by GraceWang571** [comment]
> Updated needle, see https://openqa.nue.suse.com/tests/7714601#step/rmt_chinese/64

Why the behaviors in the two verification runs are different with the same test suite?
https://openqa.nue.suse.com/tests/7714601 (In this test run, the workaround needle yast2_rmt_ssl_CA_password-Chinese-20211122 is used)
https://openqa.nue.suse.com/tests/7704169 (In this test run, no workaround needle used)

**Comment by GraceWang571** [comment]
> Updated needle, see https://openqa.nue.suse.com/tests/7714601#step/rmt_chinese/64

Why the behaviors in the two verification runs are different with the same test suite?
https://openqa.nue.suse.com/tests/7714601 (In this test run, the workaround needle yast2_rmt_ssl_CA_password-Chinese-20211122 is used)
https://openqa.nue.suse.com/tests/7704169 (In this test run, no workaround needle used)

**Comment by lemon-suse** [comment]
> > Updated needle, see https://openqa.nue.suse.com/tests/7714601#step/rmt_chinese/64
> 
> Why the behaviors in the two verification runs are different with the same test suite? https://openqa.nue.suse.com/tests/7714601 (In this test run, the workaround needle yast2_rmt_ssl_CA_password-Chinese-20211122 is used) https://openqa.nue.suse.com/tests/7704169 (In this test run, no workaround needle used)

That's for Chenzi's comments for the bsc#1127676, so I updated needles to add workaround after that.

**Comment by GraceWang571** [comment]
> > > Updated needle, see https://openqa.nue.suse.com/tests/7714601#step/rmt_chinese/64
> > 
> > 
> > Why the behaviors in the two verification runs are different with the same test suite? https://openqa.nue.suse.com/tests/7714601 (In this test run, the workaround needle yast2_rmt_ssl_CA_password-Chinese-20211122 is used) https://openqa.nue.suse.com/tests/7704169 (In this test run, no workaround needle used)
> 
> That's for Chenzi's comments for the bsc#1127676, so I updated needles to add workaround after that.

Yes, sorry for missing the comment https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13713#issuecomment-973847085.
So, I can approve this PR

**Review by GraceWang571 — APPROVED** [review]

**Review by tinawang123 — APPROVED** [review]
LGTM

**Pull request merged**
